src/homepageExperience/components/steps/arduino/ExecuteAggregateQuery.tsx

```diff
@@ -29,5 +85,19 @@ export const ExecuteAggregateQuery = (props: OwnProps) => {
     return () => document.removeEventListener('keydown', fireKeyboardCopyEvent)
   }, [])
 
-  return <h1>Execute a Flux Aggregate Query on {bucket}</h1>
+  return (
+    <>
+      <h1>Execute a Flux Aggregate Query on {bucket}</h1>
```

Comment:
I think this title is too long for a onboarding wizard, if you use the format other wizards are using, i.e. add that helper text like in the image above, you'd not need this title.